### DISCUSSION TITLE
point solutions

### DISCUSSION THREAD

**Discussion by monty3301** [CMSgov/price-transparency-guide#102] [Q&A] [answered]
If a plan uses a "point solution" provided through a separate vendor (like fertility benefits) should a file from that vendor be incorporated into the three files?  Or can that be posted separately?

**Comment by ghost** [comment]
I have heard that data that pertains to one of the 3 file types that is received from another source/vendor needs to be be interlaced with the data it is associated with in the specific plan file of that file type generated by the insurer or payor. If data was received in a JSON format or another format from a "point solution" the expectation is that data would be incorporated into the JSON file where the data found in the In Network object would be combined to seamlessly represent the data as a whole based on the CMS schema in a single file. This will be a processing intense to perform this action and it would be better if CMS provides specific guidance in this.

**Reply by vijayanandramamoorthy** [reply]
Our interpretation is,  file/data from a vendor should be on a separate file with that vendor’s name in the reporting_entity_name field of the file.
This is based on the answer from CMS for the below question
[https://github.com/CMSgov/price-transparency-guide/discussions/36](#36 )

“The intent for the reporting_entity_name element was to identify the entity that was actually making the file. Not the entity that was hosting the file (which would most likely be the issuer/payer)."

**Reply by ghost** [reply]
@vijayanandramamoorthy @monty3301 If the "point solution" was to provide additional information to the In-Network Rates JSON file, then what would be the naming convention used for the "point solution" file and you In-Network Rates JSON file that would be published on the web site for a specific month?

**Comment by shaselton-usds** [comment, marked as answer]
Wrapping some old discussions here. This can be addressed using the [table of contents file](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/table-of-contents).
